### PR TITLE
Additional data variables

### PR DESCRIPTION
**Pull request by PatFawbertMills**
Testing out the technique of adding data vars:
- As attributes where the value is rendered
- As a javascript object property on the window, e.g. `analytics.[attribute] = [value]`

### CODE REVIEW

File: src/components/PassageMatches.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const PassageMatches = ({ passages, setPassageIndex, activeIndex }: TProps) => {
       ) : (
         <div className="divide-lineBorder divide-y passage-matches-list">
           {passages.map((item, index: number) => (
-            <div key={item.text_block_id}>
+            <div key={item.text_block_id} data-analytics-passage={index + 1}>
```
Comment (diversemix) [inline src/components/PassageMatches.tsx:20]: Just need to be extra careful here as you are going from zero-based to one-based

Comment (PatFawbertMills) [reply src/components/PassageMatches.tsx:20]: This is to make it "friendly" for the analytics guys to handle

File: src/components/blocks/DocumentInfo.tsx [hunk 2]
```diff
@@ -21,6 +21,9 @@ type TDoucmentInfoProps = {
 };
 
 const DocumentInfo = ({ heading, text = "", list = [], id = "", tooltip = "", bulleted = false }: TDoucmentInfoProps) => {
+  const dataProps = {};
+  dataProps[`data-analytics-${heading.toLowerCase().trim().replace(" ", "-")}`] = "";
```
Comment (diversemix) [inline src/components/blocks/DocumentInfo.tsx:25]: Maybe worth protecting this a little more, eg having a fixed list of allowed keys somewhere to check against?

**Review by diversemix — APPROVED** [review]
lgtm

**Pull request merged**